fmt.java

```diff
@@ -0,0 +1,60 @@
+///usr/bin/env jbang "$0" "$@" ; exit $?
+//DEPS info.picocli:picocli:4.6.3
+//DEPS org.jboss.forge.roaster:roaster-jdt:2.28.0.Final
+//DEPS io.quarkus:quarkus-ide-config:2.16.3.Final
+
```

Comment:
p.s. you can add //DESCRIPTION with documentation hints that will be added to the catalog on jbang alias add and thus show up in jbang alias list